assets/js/modules/adsense/datastore/accounts.js

```diff
@@ -55,6 +55,8 @@ const baseActions = {
 			type: RESET_ACCOUNTS,
 		};
 
+		yield dispatch( STORE_NAME ).clearErrors( 'getAccounts' );
```

Comment:
Based on the IB and https://github.com/google/site-kit-wp/issues/2033#issuecomment-693193545, these calls shouldn't use `dispatch`. This applies to the other changes below as well.